events/static/css/events.less

```diff
@@ -2,24 +2,28 @@
 
 .event-list-wrapper{
     margin: 0 50px;
-
+	button{
+		width: 100%;
+	}
 }
 
 .event-item-wrapper{
 	border-bottom: 1px solid#e0e0e0;
```

Comment:
Somebody missed a space before `#`. Please add or delete entire line if not needed.